lib/idx/idxState/v1/idxResponseParser.ts

```diff
@@ -113,7 +113,18 @@ export const parseIdxResponse = function parseIdxResponse( authClient: OktaAuthI
   const remediationData = idxResponse.remediation?.value || [];
 
   remediationData.forEach(
-    remediation => expandRelatesTo(idxResponse, remediation)
+    remediation => {
+      // TODO: remove once IDX is fixed - OKTA-659181
+      if (remediation.name === 'launch-authenticator' &&
+        remediation?.relatesTo?.[0] === 'authenticatorChallenge' &&
```

Comment:
Any danger in _assuming_ it's the first and only item in `relatesTo`?

```suggestion
        remediation.relatesTo?.indexOf('authenticatorChallenge') >= 0 &&
```